website/app/styles/pages/utilities/dialog-primitive.scss

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+
+// UTILITIES > DIALOG PRIMITIVE
+
+#show-content-utilities-dialog-primitive {
+  .doc-dialog-primitive-grid-layout {
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+    background: white;
+  }
+
+  .doc-dialog-primitive-flex-layout {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    margin: 2rem;
+  }
+
+  .doc-dialog-primitive-content-placeholder {
```

Comment:
This should use a `Doc::Placeholder` component. Then you wouldn't have to add custom HTML.